app/views/pages/home.html.erb

```diff
@@ -1,2 +1,45 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(<%= asset_url 'banner.jpg'%>);">
+  <div class="container">
+    <h1>Rent-a-car <strong>All the top car hire companies in one place</strong>!</h1>
+    <p>Book your car asap</p>
+    <a class="btn btn-primary" href="#">Search for a car</a>
+  </div>
+</div>
+
+<div class="cards row">
+  <div class="card col-4" style="width: 18rem;">
```

Comment:
I don't think cards are responsive yet. You shouldn't use a specific width and there is no breakpoint modifier

Reply:
https://github.com/florian-lefebvre/airbnb-clone/pulls